languages/tgmpa-en_GB.po

```diff
@@ -0,0 +1,476 @@
+# Translation of TGM-Plugin-Activation in English (UK)
+# This file is distributed under the same license as the TGMPA package.
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM-Plugin-Activation\n"
+"POT-Creation-Date: 2016-04-21 00:07+0200\n"
+"PO-Revision-Date: 2016-04-21 14:04+0200\n"
+"Last-Translator: \n"
```

Comment:
You can put my name in here, as I was the one that translated the strings on GlotPress :-)